encoding/preparation.md

```diff
@@ -204,23 +204,14 @@ and which ones you want to use.
 
 The codec used most commonly is x264,
 an implementation of the h.264 standard.
-There are multiple versions available
-(kmod and tmod being the most popular ones),
-but they only differ slightly
-(mainly cosmetic patches and some obscure parameters
-that you will probably never use),
-and no version is objectively better than any other.
-x264 can encode in 8 or 10 bit color depth.
-We will explain the meaning of these at a later time.
-For now, they are just relevant for choosing the right binary.
-All modifications come in two versions each,
-one for 8 bit and one for 10 bit[^2].
-Most fansub group nowadays use 10 bit,
-so you should download that
-unless your group leader or encode mentor told you otherwise.
-
-- [Download kmod](http://komisar.gin.by/)
-- [Download tmod](https://github.com/jpsdr/x264/releases)
+The most recent builds can be found [here][x264vlan].
```

Comment:
It's always better to think of links in *prose* as an extension to the actual text, not as a replacement. If you say `The most recent builds can be found on [VideoLAN's site][x264vlan]`, the text can be understood without the link as you convey more information. This doesn't work all the time, but it usually does. A good rule of thumb is to never use "here" as the hyperlink text and instead either restructure the sentence or add the hyperlink to some other text in the sentence.